### PR TITLE
Update forms to v2

### PR DESCRIPTION
**Pull request by thisisdano**
[Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds/dw-forms-v2/components/detail/address-form.html)

### CODE REVIEW

File: src/stylesheets/uswds.scss [hunk 1]
```diff
@@ -48,6 +48,7 @@ Output all USWDS
 @import 'components/accordions';
 @import 'components/alerts';
 @import 'components/banner';
+@import 'components/checklist';
```
Comment (maya) [inline src/stylesheets/uswds.scss]: Isn't a checklist an input like radio buttons?

File: src/stylesheets/elements/_inputs.scss [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // scss-lint:disable QualifyingElement, PropertyCount
 
-$input-font-size-calc: scale($theme-font-family-input, $theme-font-size-base);
-$input-line-height-calc: lh($theme-font-family-input, $theme-line-height-input); // was 1.3
+$input-font-size-calc: scale($theme-font-family-forms, $theme-font-size-base);
+$input-line-height-calc: lh($theme-font-family-forms, $theme-line-height-input); // was 1.3
```
Comment (maya) [inline src/stylesheets/elements/_inputs.scss]: I have been using verbose so it's clearer to people reading the code, let's save the shortened ones for prototyping.

File: src/stylesheets/uswds.scss (hunk 1 above)
Comment (thisisdano) [reply src/stylesheets/uswds.scss]: Hm, not in this case — this is a component (in v1 as well) called `.usa-checklist` which is essentially a validation checklist of "things you need to do". Definitely open to new names for it, though.

See:
https://components.designsystem.digital.gov/components/preview/validation.html
and
https://github.com/uswds/uswds/blob/e9249a0fb47733a065a35937eda06e5a47d00f22/src/stylesheets/components/_forms.scss#L113

Comment (thisisdano) [reply src/stylesheets/uswds.scss]: It felt like it _could_ be used in more cases than just in a forms alert, though, which is why I think it should be abstracted into its own component

Comment (maya) [reply src/stylesheets/uswds.scss]: Oh, I thought this was checkbox and didn't see the connection with validation.

File: src/stylesheets/components/_alerts.scss [hunk 3]
```diff
@@ -41,6 +41,7 @@ $icon-size: units(4);
     margin-top: units(2);
   }
 
+  // TODO: why is this not simply a border?
```
Comment (maya) [inline src/stylesheets/components/_alerts.scss]: it's so you can have a plain alert with no border, try it with `usa-alert`.

Comment (thisisdano) [reply src/stylesheets/components/_alerts.scss]: Even `usa-alert` has a gray border. If there's a borderless alert, I can't track it down.  And even if there was one, the benefit of this technique (better text alignment between bordered and non-bordered alerts?) seems pretty minimal for the extra css required for an absolutely-positioned bar. 🤷‍♀️

Comment (thisisdano) [reply src/stylesheets/components/_alerts.scss]: Not saying this is a first-priority change by any means — just something I noticed

**Comment by thisisdano** [comment]
It doesn't bother me, but I could probably give it 2px — something where the resulting space is a little less than that between the label and the input

File: src/stylesheets/components/_forms.scss [hunk 4]
```diff
@@ -1,144 +1,49 @@
...
-  input {
-    margin-bottom: 3rem;
-  }
+  .usa-button {
+    margin-top: units(1);
 
-  select {
-    margin-bottom: 3rem;
+    @include at-media('mobile-lg') {
+      margin-top: units(2);
```
Comment (maya) [inline src/stylesheets/components/_forms.scss]: `units(3)` to be the same as between input and the label?

**Comment by maya** [comment]
if you use float right instead of text align on the show pw it will work.

File: src/stylesheets/components/_forms.scss (hunk 4 above)
Comment (thisisdano) [reply src/stylesheets/components/_forms.scss]: Done

**Comment by maya** [comment]
@thisisdano looks like there's a conflict. Can you resolve?

**Pull request merged**